lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,19 @@
-
 def palindrome_permutation?(string)
-  raise NotImplementedError, "palindrome_permutation? not implemented"
+  check = {}
+  string.split("").each do |i|
+    if check[i] == 1
+      check[i] += 1
+    else
+      check[i] = 1
+    end
+  end
+
+	odds = 0
+	check.values.each do |val|
+		if val % 2 != 0
+			odds += 1
+		end
+	end
```

Comment:
indentation?

Reply:
That's very odd. I have had issues with pushing from git for indentations before but never in the middle of the file. This is what it looks like in my file.
<img width="626" alt="Screen Shot 2020-04-01 at 9 26 15 PM" src="https://user-images.githubusercontent.com/58873101/78210590-7b71eb80-745f-11ea-8e26-5c8b0b39db04.png">